Pricing.md

```diff
@@ -0,0 +1,27 @@
+## Table of Contents:
+
+- [Introduction](README.md)
+- [Seal Design](Design.md)
+- [Using Seal](UsingSeal.md)
+- [Security Best Practices and Risk Mitigations](SecurityBestPractices.md)
+- [Seal Beta Terms of Service](TermsOfService.md)
+
+# Seal pricing
+
+- **Choose your own key server providers:** Seal supports a decentralized network of independent key servers. Builders can select any combination of providers to form their preferred threshold configuration for encryption and decryption.
+- **Transparent pricing & features:** Each key server provider sets their own pricing and rate limits based on their service model. Builders can evaluate and choose based on what best fits their application needs.
+- **Curated discoverability:** This documentation will list a verified set of providers along with available links to their configuration details, terms, and pricing, so you can integrate with confidence.
+
+## Verified key servers
+
+### Testnet
+
+- Mysten Labs: The following key servers are freely available for experimentation, development, and testing. A source-based rate limit is configured which can not be changed for any client.
+    - mysten-testnet-1: https://seal-key-server-testnet-1.mystenlabs.com
+    - mysten-testnet-2: https://seal-key-server-testnet-2.mystenlabs.com
```

Comment:
let's stress that we do not commit to maintain the keys forever and may change them over time